---
title: Genomic analysis of COVID-19 spread. Situation report 2020-03-26.
authors:
  - Emma Hodcroft
  - Nicola MÃ¼ller
  - Cassia Wagner
  - Misja Ilcisin
  - James Hadfield
  - Sidney M. Bell
  - Richard Neher
  - Trevor Bedford
authorLinks:
  - https://neherlab.org/emma-hodcroft.html
  - https://bedford.io/team/nicola-mueller/
  - https://bedford.io/team/cassia-wagner/
  - https://bedford.io/team/misja-ilcisin/
  - https://bedford.io/team/james-hadfield/
  - https://twitter.com/sidneymbell
  - https://neherlab.org/richard-neher.html
  - https://bedford.io/team/trevor-bedford/
affiliations: "Fred Hutch, Seattle, USA; Biozentrum, Basel, Switzerland; CZI, CA, USA"
date: "2020 XXXX XX"
dataset: "https://nextstrain.org/ncov/2020-03-26?d=map&legend=closed"
abstract: "This report uses publicly shared genomic data to track the spread of COVID-19. These reports are updated weekly."
---
<!-- Translators: Only text after : in the above ^ needs to be translated -->
<!-- Comment tags like these do not need to be translated, they are only to help you! -->
<!-- Ensure that links always end in a 'letter' (. counts) If some kind of text doesn't follow them, it breaks the slide. -->
<!-- numbers can be tagged ilke this: 161</tag> - this is just for us to help find them to update! Just leave in the </tag> bit. -->


<!-- This is left-side text -->
# [Table of Contents](https://nextstrain.org/ncov/2020-03-26?d=tree,map&p=grid)

* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=2).     
* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=3).
* [](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=X).

* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=X).
* [FAQ & common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=X).
* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=X).
<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Executive summary

We analyzed 1,495 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is evolving and moving around the world. In this week's updates, we report:

**Europe**  
* We continue to see co-mingling of samples across Europe, suggesting that the virus has continued to move across borders in the last 2-4 weeks. As mitigation measures have time to take effect, we may hope to see more clustering of cases by country.
* The North American and European outbreaks are deeply intertwined, with likely transmission in both directions.  

**North America**   
* Within the U.S., transmission patterns are complex, with samples collected from opposite sides of the country still showing close relationships.
* Washington state has had at least two independent introductions (the first from China, the second from Europe), which have led to two separate transmission chains.
* Evidence for likely local transmission within several states, most clearly within California.

**Central and South America**
* Evidence for likely local transmission within Sao Paolo, Brazil, for the last X+ days.
* Evidence for likely local transmission within Santiago, Chile, for the last X+ days.
* We received new sequences from South America this week, but sampling remains sparse; there may be much more COVID activity in this region than is captured in the available data.

**Asia**  
* Preliminary evidence for a possible second transmission chain circulating within Iran.

**Africa**  
* Evidence for likely local transmission within Kinshasa, Democratic Republic of the Congo, for the last 11+ days.
* We received new sequences from Africa this week, but sampling remains sparse; there may be much more COVID activity in this region than is captured in the available data.

**Oceania**
* Evidence for likely local transmission in New South Wales, Australia.
```


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [COVID-19 Resources](https://nextstrain.org/ncov/2020-03-26)
We've prepared some resources that are worth reading to familiarize yourself with COVID-19 and the virus that causes it, SARS-CoV-2.
This information will make interpreting the data we present in this narrative easier; if you aren't familiar with phylogenetic trees, we encourage you to check out the ['How to Read Phylogenies' narrative](https://nextstrain.org/narratives/trees-background/) and come back when you're ready.
<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

# Background and Resources  

## Nextstrain Resources  

[How to read a phylogeny](https://nextstrain.org/narratives/trees-background/)  

[Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV)

[Background on the current hCoV-19 outbreak](https://nextstrain.org/help/coronavirus/SARS-CoV-2)

[Sampling: why aren't you covering the situation in country X?](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-20?n=3)

[FAQs](https://nextstrain.org/help/coronavirus/FAQ)

[Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11)


## External Resources  
[WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports)

[CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html)

[The NYTimes COVID-19 coverage](https://www.nytimes.com/news-event/coronavirus)

[Our World in Data](https://ourworldindata.org/coronavirus)

[Ask a Scientist & FAQs](https://covid19.fas.org/l/en)

```


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [A note on sampling](https://nextstrain.org/ncov/2020-03-26?c=country&r=country&d=map&p=grid&legend=closed)
We currently have sequences from samples taken in XX countries across X continents. This is an incredible feat -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
<br><br>
While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
<br><br>
For example, the map shows very few sequences from the global south. This is NOT because COVID-19 isn't circulating in these areas, or that these cases are not as crucial to understand; rather, we just don't have much data available from these areas. The size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak.
<!-- There is NO right-side text -->




<!-- ############ EUROPE ############# -->
<!-- This is left-side text -->
# [Updates for Europe](https://nextstrain.org/ncov/2020-03-25?d=tree,map&f_region=Europe&p=grid)
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Continued spread across borders in the last 2-4 weeks](https://nextstrain.org/ncov/2020-03-26?c=country&dmin=2020-03-10&f_region=Europe)

Here, we show samples collected across Europe in the last 2 weeks, colored by country.
Notably, while there are some small clusters that are country-specific, there is also a great deal of intermingling by country.
This suggests that there has been continued spread across borders within the last 2-4 weeks.

As the recent mitigation efforts have time to take effect, we will surveil to see if cases start to cluster more discretely by country.
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [European and U.S. outbreaks are intertwined](https://nextstrain.org/ncov?c=region&label=clade:A2a&m=div)
Across the tree, we see evidence for many introductions to the U.S.
Looking within European clades like this one, we see that the origins of the U.S. and European outbreaks are deeply intertwined.

This artificially appears to flow primarily from Europe to the U.S., rather than bidirectionally; however, we expect that this is mostly an effect of sampling bias as we currently have more data from Europe.

<!-- maybe add a sentence here about how many of the US cases are not popular tourist destinations -->
<!-- There is no right side text -->




<!-- ############ NORTH AMERICA ############# -->
<!-- This is left-side text -->
# [Updates for North America](https://nextstrain.org/ncov/2020-03-26?d=tree,map&f_region=North%20America&p=grid)
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Transmission across the U.S. is complex and not well-understood](https://nextstrain.org/ncov?f_division=Connecticut&label=clade:B1&m=div)

Even within the U.S., transmission patterns are highly complex.

For example, recent samples from the Grubaugh lab in Connecticut are both ancestral to and intermingled with samples from Washington state.
While more data is needed to draw firm conclusions, this suggests that there has been recent transmission between even very distant parts of the country.

This highlights that the transmission patterns across the U.S. are far more complex than is represented in the currently available data.
Additional data will help us understand what is likely a deeply interconnected national epidemic playing out amidst highly heterogeneous responses across states.
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Multiple transmission chains circulating in Washington state](https://nextstrain.org/ncov/2020-03-26?d=tree,map&f_country=USA&f_division=Washington&p=full&r=location)

Multiple introductions to Washington state have sparked at least two local transmission chains / clusters of cases.
The first was most likely imported from China.
The second was most likely of European origin.
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Likely two local transmission chains in California](https://nextstrain.org/ncov/2020-03-26?branchLabel=clade&c=location&f_country=USA&f_division=California&label=clade:A7&p=full)

Here, we see several samples from Solano County, California, that are sampled on two different dates but very closely related to one another.

While the available data from California is too limited to make firm conclusions, this preliminary evidence is consistent with local transmission within Solano County.

Elsewhere in the tree, we see a similar pattern for samples in Santa Clara county.

Each county's cluster of cases is likely to due to a separate introduction, based on their relative positions in the tree.
<!-- There is no right side text -->


<!-- ############ SOUTH AMERICA ############# -->
<!-- This is left-side text -->
# [Updates for Central & South America](https://nextstrain.org/ncov/2020-03-26?d=tree,map&f_region=Central%20America,South%20America&p=grid)
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Primarily travel-related cases sampled in South America](https://nextstrain.org/ncov/2020-03-26?d=tree,map&f_country=Chile&label=clade:B&m=div&p=full)
We don't have much data from South America yet; of those data, most are returned travelers.

While there are some potentially interesting patterns starting to emerge in the non-travel-associated cases, we don't yet have enough evidence to draw conclusions.
Sequencing efforts are ramping up, and we will report back as the situation becomes clearer.
<!-- There is no right side text -->




<!-- ############ ASIA ############# -->
<!-- This is left-side text -->
# [Updates for Asia](https://nextstrain.org/ncov/2020-03-26?d=tree,map&f_region=Asia&p=grid)
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Possible second transmission chain within Iran](https://nextstrain.org/ncov?c=division_exposure&label=clade:A3&m=div)
Pakistan sequence is basal to the (inferred genomic position of the) Iranian outbreak
<!-- There is no right side text -->

We do not yet have viral genomes from Iran.
However, we do have sequences from many cases with reported travel to Iran.

Most of the Iranian-linked cases nest within this clade.

At the very bottom, we see a new sequence from a woman in Pakistan with reported travel to Iran.
Interestingly, this case is an outgroup to all of the other cases that we know about from Iran.

This is consistent with a possible second transmission chain within Iran, although more samples are needed to confirm.
<!-- There is no right side text -->




<!-- ############ AFRICA ############# -->
<!-- This is left-side text -->
# [Updates for Africa](https://nextstrain.org/ncov/2020-03-26?d=tree,map&f_region=Africa&p=grid)
<!-- There is no right side text -->

<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Evidence for local transmission in Kinshasa, Congo](https://nextstrain.org/ncov/2020-03-26?d=tree,map&f_country=Congo&f_region=Africa&label=clade:A2a&p=grid)

New sequences from Kinshasa in the Democratic Republic of the Congo, generated by Placide Mbala-Kingebeni et al., form a group of very closely related, local cases that were sampled over several days.

This is consistent with local transmission, ongoing since at least March 15 (the sampling date of the earliest case that appears to be part of this local transmission chain) and possibly earlier.

Importantly, this does not necessarily indicate that the outbreak is more advanced in the DRC than elsewhere in Africa; rather, this very likely represents uneven sampling across the continent.
<!-- There is no right side text -->




<!-- ############ OCEANIA ############# -->
<!-- This is left-side text -->
# [Updates for Oceania](https://nextstrain.org/ncov/2020-03-25?d=tree,map&f_region=Oceania&p=grid)
<!-- There is no right side text -->


<!-- ##### slide break ##### -->
<!-- This is left-side text -->
# [Local transmission in New South Wales](https://nextstrain.org/ncov/2020-03-26?Label=clade:A3&d=tree&f_division=New%20South%20Wales&label=clade:A3&p=full&r=location)

New sequences from Australia provide evidence of local transmission across New South Wales.
The virus has likely been circulating in this area since at least February 29, 2020 (the sampling date of Australia/NSW07/2020).

<!-- There is no right side text -->




<!-- ##### slide break ##### -->
<!-- This is left-side text 13-->
# [Takeaways](https://nextstrain.org/ncov/2020-03-20?c=country&d=map&p=full)
-  In the next few weeks, we will be watching the European outbreak for signs that the new mitigation measures have stemmed cross-border transmission.
<br>
-  The U.S. epidemic likely has very complex transmission patterns of both import and spread across even disparate parts of the country. Testing and sequencing are needed urgently.
<br>
- We find evidence of local transmission in New South Wales, Australia.
<br>
- We find evidence of local transmission in Kinshasa in the DRC. We do not yet have enough data to assess whether the virus is circulating locally in other parts of Africa.
<br>
- We will be monitoring the situation in South America as more sequences become available.
<br>
- Controlling local outbreaks through social distancing is crucial to:  
  - #FlattenTheCurve to prevent healthcare systems from being overwhelmed by many simultaneous cases  
  - Reduce the total number of cases and fatalities  
  - Allow time for the development of therapeutics and vaccines  

<!-- This is the right-side text -->

```auspiceMainDisplayMarkdown
# What you can do
## ...as an individual
* Dramatically reduce the number of people you are in contact with each day, especially if you are in a vulnerable group (e.g., seniors and those with pre-existing conditions).
* Remember that even if you are not super vulnerable, many people around you are; follow these practices to protect others.
* Wash your hands "like you just chopped a jalapeno and have to change a contact lens."  
* Stay home as much as possible -- especially  if you are sick; be prepared with extra supplies in case you need to self-quarantine.  
* If you are an employer, encourage your employees to work from home wherever possible

## ...as an official  
* Make testing free and broadly available.  
* Put strong social distancing measures in place.  
* Fund and implement extensive contact tracing efforts.  
* Financially support those impacted by social distancing measures (e.g., hourly workers, those with elder or childcare responsibilities, small businesses, etc.).
```

<!-- ##### slide break ##### -->






<!-- ##### slide break ##### -->

<!-- This is left-side text -->
# [Scientific credit](https://nextstrain.org/ncov/2020-03-26?d=map&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak, and particularly those working in China.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.

<br>

We also gratefully acknowledge [GISAID](https://gisaid.org) for providing the platform through which these data can be uploaded and shared.

<!-- Do not need to translate institutions names -->
<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

We are grateful for the data gathered by these originating labs:



```


<!-- ##### slide break ##### -->

<!-- This is left-side text -->
# [Detailed scientific credit](https://nextstrain.org/ncov/2020-03-05?d=map&c=author)

These data were shared via [GISAID](https://gisaid.org).
We gratefully acknowledge their contributions.

<br>

To the right we give specific sequences shared by each lab.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

The SARS-CoV-2 genomes were generously shared by scientists at these submitting labs:




```
